data/src/main/scala/cats/data/Or.scala

```diff
@@ -381,6 +374,10 @@ sealed abstract class OrInstances extends OrInstances0 {
       def empty =
         ROr(Monoid[B].empty)
     }
+
+  implicit def OrdShow[A: Show, B: Show]: Show[A Or B] =
```

Comment:
good catch for the typo, I will update it.
You're right consistency is important, I will refactor to whatever we decide to go to.